src/app/pages/settings-integration/pricing/simple/settings-simple-pricing.component.html

```diff
@@ -11,13 +11,15 @@ <h3 class="card-title">{{'settings.pricing.settings.title' | translate}}</h3>
       <div class="col-md-6">
         <mat-form-field>
           <mat-label for="inputPriceUnit">{{'pricing.price_unit_label' | translate}} *</mat-label>
-          <mat-select id="inputPriceUnit" [formControl]="currency">
+          <mat-select id="inputPriceUnit"
+            [disabled]="isCurrencyCodeReadonly"
```

Comment:
Violation of golden rule 36:
<img width="854" alt="2022-01-24_17-41-37" src="https://user-images.githubusercontent.com/25266945/150825801-6d54dc7a-719e-44fc-9084-79970f2fb8a4.png">